### DISCUSSION TITLE
Custom properties with v2rc1

### DISCUSSION THREAD

**Discussion by thenitai** [plausible/analytics#2984] [Self-Hosted Support] [answered]
Hi,

I've installed the latest v2rc1 to get the custom page view properties to appear (the previously installed v1.5.1 did not show them).

However, they don't work, or maybe I do something wrong. This is how I pass the custom props:

```
const { trackPageview } = Plausible({ ...config... });
trackPageview({}, {
	props: { "id": '(value)' }
})
```

I expected to see a "Property" value in the filter dropdown. Statistics work as expected.

Any help would be appreciated.

**Comment by ruslandoga** [comment, marked as answer]
👋 @thenitai 

The custom properties filter seems to be hidden behind a feature flag. Can you please try enabling it manually with

```console
$ cd hosting
$ docker compose exec plausible bin/plausible rpc 'FunWithFlags.enable(:custom_dimension_filter)'
```

**UPDATE:** this flag is removed in `v2.0.0-rc.2` https://github.com/plausible/analytics/discussions/3011, so the custom properties filter is automatically enabled there.

**Reply by ruslandoga** [reply]
PR that removes the feature flag: https://github.com/plausible/analytics/pull/2996

**Reply by thenitai** [reply]
Thanks. That worked perfectly. Finally, Plausible is really the tool I was looking for :)

**Comment by thenitai** [comment]
Quick question (now that properties work). It looks like I cannot create sites with different filters, e.g.:

- site1.domain.com -> site 1
- site2.domain.com -> site 1
- siteA.domain.com -> site 2
- siteB.domain.com -> site 2
- *.domain.com -> site 3

Would you recommend to put everything under one site and then using the properties (URL field) to differentiate?

Thanks again.

**Reply by ruslandoga** [reply]
Sorry, I don't know how to best approach this as I haven't used this feature myself yet.
